syne_tune/experiments.py

```diff
@@ -144,13 +146,15 @@ def load_experiment(
             results = pd.read_csv(path / "results.csv")
     except Exception:
         results = None
-    try:
-        tuner = Tuner.load(path)
-    except FileNotFoundError:
-        tuner = None
-    except Exception:
+    if load_tuner:
+        try:
+            tuner = Tuner.load(path)
+        except FileNotFoundError:
```

Comment:
Is this needed?

Reply:
There may be cases where you cannot load the tuner but still want the results.